src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+function isMarried(person) {
+  if (person.isMarried) {
+    return person.sex === 'm'
+      ? `${person.partnerName} is my wife`
+      : `${person.partnerName} is my husband`;
+  }
+
+  return 'I am not married';
+}
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age ? <p className="Person__age">I am {person.age}</p> : false}
```

Comment:
The conditional rendering of the age uses `false` as the fallback value. In React, it's more common to use `null` instead of `false` for elements that should not be rendered. Consider changing `false` to `null` for better readability and convention.